samples/basic/cards/nodejs/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cards-bot",
+  "name": "node-cards-bot",
   "version": "1.0.0",
   "private": true,
   "description": "Agents cards bot sample",
```

Comment:
The name is odd. Is this an Agent or a bot?

Reply:
It's both !! we are using _EchoBot_ everywhere